binscripts/rvm-installer

```diff
@@ -793,17 +793,22 @@ rvm_install_validate_volume_mount_mode()
   do
       if [[ -d $path ]]
       then
+        test_exec=$(mktemp $path/rvm-exec-test.XXXXXX)
+        echo '#!/bin/sh' > "$test_exec"
+        chmod +x "$test_exec"
         partition=`df -P $path | awk 'END{print $1}'`
 
-        if mount | grep "$partition .*noexec.*" >/dev/null 2>&1
+        if ! "$test_exec"
         then
+          rm -f "$test_exec"
           printf "%b" "
 It looks that partition holding RVM destination location ${rvm_path}
 is mounted in *noexec* mode, which prevents RVM from working correctly.
 Please re-mount partition ${partition} without the noexec option."
```

Comment:
+1 from me, FWIW. The partition can be mounted in several places at once with different flags (which is what actually caused issues for me in the first place).

I can fix that but call me stupid but I don't know how to update the pull request without dropping changes by @pkuczynski